src/main/java/frc/robot/subsystems/ColorPanelSubsystem.java

```diff
@@ -0,0 +1,148 @@
+package frc.robot.subsystems;
+
+import com.revrobotics.ColorSensorV3;
+import com.revrobotics.ColorMatchResult;
+import com.revrobotics.ColorMatch;
+import edu.wpi.first.wpilibj.I2C;
+import edu.wpi.first.wpilibj.smartdashboard.SmartDashboard;
+import edu.wpi.first.wpilibj.util.Color;
+import frc.robot.posev2.WheelColorValue;
+import edu.wpi.first.wpilibj.DriverStation;
+
+public class ColorPanelSubsystem extends BaseSubsystem{
```

Comment:
None of the code in here can be tested. We need to move a lot of it to a separate class.

I'd recommend a Color